### PR TITLE
I30 igvc msgs

### PR DESCRIPTION
**Pull request by wkhademi**
- Created meta package to hold common messages package.
- Created a few necessary messages that will be used by other IGVC developed packages.

### CODE REVIEW

File: src/messages_pkg/igvc_msgs/msg/TrajectoryPoint.msg [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Header header
+
+# a [x,y,z] position in the global coordinate frame
+Point point
+
+# a scalar representing the longitudinal velocity
+float64 velocity
```
Comment (ssmarok) [inline src/messages_pkg/igvc_msgs/msg/TrajectoryPoint.msg]: May want to use a Vector3 for velocity for consistency.

**Review by ssmarok — APPROVED** [review]
Looks good! Just a minor comment about TrajectoryPoint velocity.

File: src/messages_pkg/igvc_msgs/msg/TrajectoryPoint.msg (hunk 1 above)
Comment (wkhademi) [reply src/messages_pkg/igvc_msgs/msg/TrajectoryPoint.msg]: Updated TrajectoryPoint velocity to Vector3 in latest commit.

Comment (ssmarok) [reply src/messages_pkg/igvc_msgs/msg/TrajectoryPoint.msg]: Looks good!

**Pull request merged**